<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="0.180" timestamp="2024-02-12T18:34:54.777478" hostname="CO-IT021519"><testcase classname="" name="tests.model.test_model" time="0.000"><error message="collection failure">ImportError while importing test module '/Users/cristhian.plazas/Desktop/Projects/latam/tests/model/test_model.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
__init__.py:1: in &lt;module&gt;
    from challenge.api import app
E   ModuleNotFoundError: No module named 'challenge'</error></testcase></testsuite></testsuites>